### ISSUE TITLE
Request: Desktop Version

### ISSUE THREAD

**Issue by 94b42c29dd** [Telegram-FOSS-Team/Telegram-FOSS#296]
Would love to see Telegramdesktop-FOSS for GNU/Linux. What would it take to make that happen?

**Comment by dm17** [comment]
> tdesktop is already FOSS, packaged in many popular distributions :) what do you mean? rofl

Right, tdesktop is FOSS - so curious what the differences are between them:
https://github.com/Telegram-FOSS-Team/Telegram-FOSS
and
https://github.com/telegramdesktop/tdesktop
